capture.sh

```diff
@@ -11,9 +20,28 @@ else
   exit 1
 fi
 
+PERIODI=4 #default to 4 seconds between captures. Can be changed with --period command line option
```

Comment:
Nit: space after comment

```suggestion
PERIODI=4 # default to 4 seconds between captures. Can be changed with --period command line option
```